Campaign update all the things.

angaither opened the pull request:
- match fields to pantheon form
- update group orders
- add helper text

angaither committed:
Campaign update all the things.
- match fields to pantheon form
- update group orders
- add helper text

aaronschachter commented:
Let's walk through this PR tomorrow.  Some thoughts:
- A field name I'd suggest changing is `confirmation_sentence`.  At first I figured this would be the msg appears when you sign up, but instead its reportback.  Possibly change that to something like `reportback_confirm_msg` so we could also have a `signup_confirm_msg` if we ever need one.
- Also unsure of what the baby_steps fields are for.

angaither committed:
Updating dosomething_campaign node type.
- removing baby steps, adding step pre/post header/copy
- rename confirmation message

angaither committed:
Updated image field names on campaign node.

aaronschachter commented:
:+1:
